FishyTransport/Sources/FishyActorsGenerator/main.swift

```diff
@@ -44,14 +41,23 @@ struct FishyActorsGeneratorMain: ParsableCommand {
       print("Generate extensions...")
     }
 
-    let sourceGen = SourceGen(targetDirectory: targetDirectory, buckets: buckets)
+    let sourceGen = SourceGen(buckets: buckets)
+    
     for decl in analysis.decls {
       if verbose {
         print("  Generate 'FishyActorTransport' extensions for 'distributed actor \(decl.name)' -> \(targetFilePath(targetDirectory: targetDirectory, i: 1))")
       }
-      _ = sourceGen.generate(decl: decl)
+      for (bucketNum, bucket) in sourceGen.generate(decl: decl).enumerated() {
```

Comment:
I commented above but this is somewhat wrong, it's not that every decl emits many sources which are many buckets; every single decl goes into a _specific_ bucket and must always go to the same bucket; 

(this loop is incorrect)

Reply:
ok that makes sense, I admit I glossed over the details of how bucketing would work.